modules/reference/pages/systems/bootstrapping.adoc

```diff
@@ -20,8 +20,20 @@ User::
 Input the clients user login.
 The default is {rootuser}.
 
+// I've added these new bits in the existing format for now, but this whole section will get overhauled with the Ref Guide updates. LKB --2020-04-17
+Authentication Method::
+Select either ``Password`` or ``SSH Private Key``.
+
 Password::
-Input the clients login password.
+For password authentication, enter the client's login password.
+
+SSH Private Key::
+For SSH authentication, copy the SSH private key.
```

Comment:
```suggestion
For SSH key authentication, copy the SSH private key.
```
As mentioned in the changelog, I'd be explicit about the key here (the "SSH authentication" happens also in the password case).